discord/message.py

```diff
@@ -637,7 +637,13 @@ def jump_url(self):
     @property
     def quote(self):
         """:class:`str`: Returns a string formatted as a quote, without the author mention."""
-        return "\n".join(["> {line}".format(line) for line in self.content.splitlines()])
+        if self.type is MessageType.default:
+            text = self.clean_content
+        else:
+            text = self.system_content
+
+        return "\n".join(["> {line}".format(line) for line in text.splitlines()])
```

Comment:
if you're gonna do this in a fairly unnecessary way at least make sure you specify the kwarg in that format 😔

Reply:
Oops